### PR TITLE
[#1977] improvment(dashboard): Add version and git commit id in dashboard

### PR DESCRIPTION
**Pull request by lwllvyb**
### What changes were proposed in this pull request?
Display version and git commit id in dashboard

### Why are the changes needed?

The version and git commit id can be used to easily locate whether the component has been upgraded and troubleshoot problems.

### Does this PR introduce _any_ user-facing change?

User can view version and git commit id from dashboard

### How was this patch tested?

Use the latest version, open the dashboard ui, and you can see the version and git commit id on the Coordinator and Shuffle server pages.
Submit a task, and you can see the version and git commit id of each application task in the Apps module on the Application page.

### CODE REVIEW

**Comment by rickyma** [comment]
PTAL @yl09099 @maobaolong

**Comment by rickyma** [comment]
@lwllvyb Tests failed. Please fix it.

File: proto/src/main/proto/Rss.proto [hunk 1]
```diff
@@ -391,6 +393,8 @@ message AppHeartBeatResponse {
 message ApplicationInfoRequest {
   string appId = 1;
   string user = 2;
+  optional string version = 3;
```
Comment (jerqi) [inline proto/src/main/proto/Rss.proto:396]: Why  does ApplictionInfoRequest need `version` and `gitCommitId`?

Comment (lwllvyb) [reply proto/src/main/proto/Rss.proto:396]: With version and git commit id, we can quickly get the client branch information, which can help us quickly locate the problem.

**Comment by jerqi** [comment]
Could you create an issue for this pull request?

**Comment by maobaolong** [comment]
@lwllvyb Maybe you can add some screenshot to show the UI.

**Comment by yl09099** [comment]
The list is getting more and more columns, and it's really not pretty.

**Comment by lwllvyb** [comment]
> Could you create an issue for this pull request?

I create an issue for this PR. 
#1977

**Review by jerqi — APPROVED** [review]
LGTM.

**Comment by jerqi** [comment]
@yl09099 Do you have another suggestion?

**Comment by lwllvyb** [comment]
ping @jerqi @yl09099

**Comment by yl09099** [comment]
Change the table to a dynamic column display, and display related columns according to your needs.

**Comment by lwllvyb** [comment]
> Change the table to a dynamic column display, and display related columns according to your needs.

Your suggestions are excellent. I don't have much experience with Vue, and what I've done so far is the extent of what I can manage at the moment。

**Comment by maobaolong** [comment]
@lwllvyb You can paste the what ui like to the description, so @yl09099 can take a look.

**Comment by lwllvyb** [comment]
This picture for Coordinator page.
image
This picture for Shuffle Server page.
image
This picture for Application page.
image

**Review by rickyma — APPROVED** [review]
LGTM

**Review by maobaolong — APPROVED** [review]
LGTM @lwllvyb Thanks for this great contribution!

**Comment by rickyma** [comment]
@lwllvyb There are conflicts need to be fixed.

@xianjingfeng Would you take another look?

**Comment by lwllvyb** [comment]
image
One of the checks failed, but the error doesn't seem to be caused by my changes. What should I do?
@rickyma @jerqi @maobaolong

**Comment by maobaolong** [comment]
@lwllvyb This is a bad flaky test which I encountered many time, you can ignore this fail test, of you can close and reopen this PR to trigger re-check if you want all checks green.

**Review by rickyma — APPROVED** [review]

**Comment by rickyma** [comment]
New conflicts left to be resolved.

**Comment by lwllvyb** [comment]
> New conflicts left to be resolved.

Fixed

**Review by rickyma — APPROVED** [review]

**Comment by rickyma** [comment]
Merged. Thanks for your contribution @lwllvyb

**Pull request merged**